README.md

```diff
@@ -24,17 +24,30 @@ Download Helm dependencies:
 
 Temporal can be configured to run with a couple of database choices.
 
-#### Cassandra and ElasticSearch
+#### Default Installation: Batteries Included
 
-By default, Temporal Helm Chart configures Temporal to runs with Cassandra (for persistence) and ElasticSearch (for "visibility" features).
+By default, Temporal Helm Chart configures Temporal to runs with Cassandra (for persistence) and ElasticSearch (for "visibility" features). By default, Temporal Helm Chart installs all the dependencies, out of the box.
 
 To install Temporal with all of its dependencies, including Cassandra and ElasticSearch, run this command:
 
 ```bash
 ~/temporal-helm$ helm install temporaltest . --timeout 900s
 ```
 
-#### MySQL Running Separately
+#### Bring Your Own ElasticSearch
+
+You might already be operating an instance of ElasticSearch that you want to use with Temporal.
+
+To do so, fill in the relevant configuration values in `values/values.cassandra.yaml`, and pass the file to 'helm install'.
+
+Exmaple:
```

Comment:
typo here

Reply:
thank you! fixed creative spelling here. ;)